scripts/localchat.lic

```diff
@@ -0,0 +1,344 @@
+=begin
+  localchat.lic: Creates a separate window to isolate chat messages you care about
+
+        author: Eleazzar
+          game: Gemstone
+          tags: chat
+       version: 2023.07.01
+      required: gtk3
+=end
```

Comment:
Header needs a slight update to align with elanthia-online hosting. Also putting `gtk3` as required doesn't work per Lich required header check. Would recommend just checking Lich > v5. Also recommend moving to semantic versioning. Suggested replacement header below:

```ruby
=begin
  localchat.lic: Creates a separate window to isolate chat messages you care about

        author: elanthia-online
  contributors: Eleazzar
          game: Gemstone
          tags: chat
      required: Lich >= 5.0
       version: 1.0.0

     Help Contribute: https://github.com/elanthia-online/scripts

  Version Control:
    Major_change.feature_addition.bugfix
  v1.0.0 (2023-07-01)
    - initial release

=end
```